include/ts/apidefs.h.in

```diff
@@ -147,959 +157,1360 @@ extern "C" {
 #endif
 
 #ifndef TS_DEPRECATED
-#define TS_DEPRECATED __attribute__((deprecated))
+#define TS_DEPRECATED [[deprecated]]
```

Comment:
We should probably remove this and do the substitution. This was done initially because of OS dependencies, but now it's part of C++.

Reply:
Whacked.